context/MqlContext/MqlContextProvider.tsx

```diff
@@ -21,6 +21,7 @@ type Props = {
   externalConfig?: {
     mqlServerUrl?: string;
     refetchMqlServerUrl?: () => void;
+    headers?: Record<string, string>;
```

Comment:
Can you make this more descriptive?  So we know what headers are allowed?

Maybe
```
{ modelId: string }
```

Reply:
I'm doing this on purpose so that we can allow pass arbitrary headers to a http client, which is a pretty standard function provided by most of the client library :)

Reply:
Makes since to want to keep this flexible, but it doesn't give future devs any insight into what headers are being passed.  It looks like this type needs to be shared though, perhaps create a new global types file for this package and then import the Headers type into the web package.  The web package can then use this type as the base type for it's headers.